clients/cobol-lsp-vscode-extension/jest.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
       "<rootDir>/dist",
     ],
     "collectCoverage": true,
+    "coveragePathIgnorePatterns":["<rootDir>/src/_tests_/*.ts"],
```

Comment:
Have you tested in SonarCloud if this folder really excluded?

Reply:
Looked into it now. It doesn't exclude.